manager/controllers/app/select_modules.go

```diff
@@ -162,7 +162,7 @@ func (p *PlotterGenerator) validate(item *DataInfo, solution Solution, appContex
 	// Are all capabilities that need to be deployed supported in this data path?
 	supportedCapabilities := []string{}
 	for _, element := range solution.DataPath {
-		supportedCapabilities = append(supportedCapabilities, element.Module.Spec.Capabilities[element.CapabilityIndex].Capability)
+		supportedCapabilities = append(supportedCapabilities, string(element.Module.Spec.Capabilities[element.CapabilityIndex].Capability))
 	}
```

Comment:
Why are the values of type string in the first place? Also, maybe unrelated to this PR but why a list and not a map? (`map[taxonomy.Capability]bool`).

Reply:
changed